data/sql_setup/prepare_schedule_b.sql

```diff
@@ -130,16 +329,72 @@ $$ language plpgsql;
 -- Create new triggers
 drop trigger if exists nml_sched_b_after_trigger on disclosure.nml_sched_b;
 create trigger nml_sched_b_after_trigger after insert or update
-    on disclosure.nml_sched_b for each row execute procedure ofec_sched_b_insert_update_queues(:START_YEAR_AGGREGATE);
+    on disclosure.nml_sched_b for each row execute procedure ofec_sched_b_insert_update(:START_YEAR_AGGREGATE);
 
 drop trigger if exists nml_sched_b_before_trigger on disclosure.nml_sched_b;
 create trigger nml_sched_b_before_trigger before delete or update
-    on disclosure.nml_sched_b for each row execute procedure ofec_sched_b_delete_update_queues(:START_YEAR_AGGREGATE);
+    on disclosure.nml_sched_b for each row execute procedure ofec_sched_b_delete_update(:START_YEAR_AGGREGATE);
 
 drop trigger if exists f_item_sched_b_after_trigger on disclosure.f_item_receipt_or_exp;
 create trigger f_item_sched_b_after_trigger after insert or update
-    on disclosure.f_item_receipt_or_exp for each row execute procedure ofec_sched_b_insert_update_queues(:START_YEAR_AGGREGATE);
+    on disclosure.f_item_receipt_or_exp for each row execute procedure ofec_sched_b_insert_update(:START_YEAR_AGGREGATE);
 
 drop trigger if exists f_item_sched_b_before_trigger on disclosure.f_item_receipt_or_exp;
 create trigger f_item_sched_b_before_trigger before delete or update
-    on disclosure.f_item_receipt_or_exp for each row execute procedure ofec_sched_b_delete_update_queues(:START_YEAR_AGGREGATE);
+    on disclosure.f_item_receipt_or_exp for each row execute procedure ofec_sched_b_delete_update(:START_YEAR_AGGREGATE);
+
+CREATE OR REPLACE FUNCTION increment_aggregates(view_row fec_fitem_sched_b_vw) RETURNS VOID AS $$
```

Comment:
One addition remark:  This will be okay sharing the same name as the schedule A version?  I know the whole method signature is different, but I just want to confirm my understanding is correct.

Same remarks as above with checking on the calculations, too.

Reply:
Good catch! I'll review the namespace and overloading features in `plpgsql` and report.

Reply:
It is permitted and they will not conflict - https://www.postgresql.org/docs/9.0/static/sql-createfunction.html. However, as it is subtle and we do not need overloading, I will rename the functions to be explicit.

Reply:
Thanks, that sounds like a good idea!